plugin-mapping.properties

```diff
@@ -135,4 +135,5 @@ seatunnel.sink.MongoDB = connector-mongodb
 seatunnel.source.Iceberg = connector-iceberg
 seatunnel.source.InfluxDB = connector-influxdb
 seatunnel.source.S3File = connector-file-s3
-seatunnel.sink.S3File = connector-file-s3
\ No newline at end of file
+seatunnel.sink.S3File = connector-file-s3
+seatunnel.source.MyHours = connector-http-myhours
```

Comment:
Need to sync update `plugin_config`?


https://github.com/apache/incubator-seatunnel/blob/dev/config/plugin_config

Reply:
Thanks for your review, I have fixed it.